src/tests/pintest.c

```diff
@@ -64,7 +65,9 @@ static int ask_and_verify_pin(struct sc_pkcs15_object *pin_obj)
 		return 0;
 	}
 
-	sprintf(prompt, "Please enter PIN code [%.*s]: ", (int) sizeof pin_obj->label, pin_obj->label);
+	snprintf(prompt, sizeof(prompt), "Please enter PIN code [%.*s]: ",
+		(int) MIN(sizeof(prompt) - 26, strlen(pin_obj->label)),
```

Comment:
shouldn't we use `(int) MIN(sizeof(pin_obj->label), strlen(pin_obj->label)`?

Reply:
the label is here again static buffer, so I am not sure how the `sizeof()` could help us here.